dll/win32/shell32/wine/PolicyData.h

```diff
@@ -0,0 +1,232 @@
+DEFINE_POLICY( REST_NOSHELLSEARCHBUTTON,         Explorer,       NoShellSearchButton             )
+DEFINE_POLICY( REST_NOHARDWARETAB,               Explorer,       NoHardwareTab                   )
+DEFINE_POLICY( REST_NORUNASINSTALLPROMPT,        Explorer,       NoRunasInstallPrompt            )
+DEFINE_POLICY( REST_PROMPTRUNASINSTALLNETPATH,   Explorer,       PromptRunasInstallNetPath       )
+DEFINE_POLICY( REST_NOMANAGEMYCOMPUTERVERB,      Explorer,       NoManageMyComputerVerb          )
+#if (NTDDI_VERSION < NTDDI_VISTA)
+DEFINE_POLICY( REST_NORECENTDOCSNETHOOD,         Explorer,       NoRecentDocsNetHood             )
+#endif
+DEFINE_POLICY( REST_DISALLOWRUN,                 Explorer,       DisallowRun                     )
+DEFINE_POLICY( REST_NOWELCOMESCREEN,             Explorer,       NoWelcomeScreen                 )
+DEFINE_POLICY( REST_RESTRICTCPL,                 Explorer,       RestrictCpl                     )
+DEFINE_POLICY( REST_DISALLOWCPL,                 Explorer,       DisallowCpl                     )
+DEFINE_POLICY( REST_NOSMBALLOONTIP,              Explorer,       NoSMBalloonTip                  )
+DEFINE_POLICY( REST_NOSMHELP,                    Explorer,       NoSMHelp                        )
+DEFINE_POLICY( REST_NOWINKEYS,                   Explorer,       NoWinKeys                       )
+DEFINE_POLICY( REST_NOENCRYPTONMOVE,             Explorer,       NoEncryptOnMove                 )
+DEFINE_POLICY( REST_NOLOCALMACHINERUN,           Explorer,       DisableLocalMachineRun          )
+DEFINE_POLICY( REST_NOCURRENTUSERRUN,            Explorer,       DisableCurrentUserRun           )
+DEFINE_POLICY( REST_NOLOCALMACHINERUNONCE,       Explorer,       DisableLocalMachineRunOnce      )
+DEFINE_POLICY( REST_NOCURRENTUSERRUNONCE,        Explorer,       DisableCurrentUserRunOnce       )
+DEFINE_POLICY( REST_FORCEACTIVEDESKTOPON,        Explorer,       ForceActiveDesktopOn            )
+#if (NTDDI_VERSION < NTDDI_VISTA)
+DEFINE_POLICY( REST_NOCOMPUTERSNEARME,           Explorer,       NoComputersNearMe               )
+#endif
+DEFINE_POLICY( REST_NOVIEWONDRIVE,               Explorer,       NoViewOnDrive                   )
+#if (NTDDI_VERSION >= NTDDI_WINXP) || defined(IE_BACKCOMPAT_VERSION)
+DEFINE_POLICY( REST_NONETCRAWL,                  Explorer,       NoNetCrawling                   )
+DEFINE_POLICY( REST_NOSHAREDDOCUMENTS,           Explorer,       NoSharedDocuments               )
+#endif
+DEFINE_POLICY( REST_NOSMMYDOCS,                  Explorer,       NoSMMyDocs                      )
+#if (NTDDI_VERSION >= NTDDI_WINXP)
+DEFINE_POLICY( REST_NOSMMYPICS,                  Explorer,       NoSMMyPictures                  )
+DEFINE_POLICY( REST_ALLOWBITBUCKDRIVES,          Explorer,       RecycleBinDrives                )
+DEFINE_POLICY( REST_NONLEGACYSHELLMODE,          Explorer,       NoneLegacyShellMode             )
+DEFINE_POLICY( REST_NOCONTROLPANELBARRICADE,     Explorer,       NoControlPanelBarricade         )
```

Comment:
```suggestion
DEFINE_POLICY( REST_NOCONTROLPANELBARRICADE,     Explorer,       NoControlPanelBarricade         )
// NOTE: REST_NOSTARTPAGE never really existed.
```
I'm adding back this comment, because this NOSTARTPAGE thing is still documented in MSDN, even though it never appears in this table (in shell32). So it's to make any dev looking at here, understand why it's not appearing here.